client/tooltips.js

```diff
@@ -0,0 +1,156 @@
+    
+    annotation: "An optional note about a rule."
+  },
+  sbmlComponentsEditor: {
+    annotation: "An optional note about the SBML Component."
+  },
+  modelSettings: {
+    previewSettings: "Preview Settings are applied to the model and are used for the model preview and all workflows.",
+    
+    previewTime: "End time of simulation.",
+    
+    timeUnits: "Save point increment for recording data.",
+    
+    volume: "The volume of the system matters when converting to from population to concentration form. This will also set a parameter 'vol' for use in custom (i.e. "+
+        "non-mass-action) propensity functions."
+  },
+  simulationSettings: {
+    rtol: "Relative tolerance for ode solutions, controls a relative accuracy (number of correct digits).  The solver keeps the local error estimates less than atol + "+
+        "rtol * abs(y).  Value must be greater than 0.0.",
+
+    atol: "Absolute tolerance for ode solutions, if a component of y is approximately below atol, the error only needs to fall within the same atol threshold, and the "+
+        "number of correct digits is not guaranteed.  The solver keeps the local error estimates less than atol + rtol * abs(y).  Value must be greater than 0.0.",
+
+    ttol: "A relative error tolerance value governing tau-leaping tau selections.  Based on Cao, Y.; Gillespie, D. T.; Petzold, L. R. (2006). 'Efficient step size "+
+        "selection for the tau-leaping simulation method' (PDF). The Journal of Chemical Physics. 124 (4): 044109. Bibcode:2006JChPh.124d4109C. doi:10.1063/1.2159468. "+
+        "PMID 16460151  Value must be between 0.0 and 1.0.",
+
+    seed: "The seed for the simulation.  Set to -1 for a random seed.",
+
+    realizations: "The number of times to sample the chemical master equation. Each trajectory will be returned at the end of the simulation.",
+
+    algorithm:"The solver by which to simulate the model.",
+
+    ode: "This algorithm produces a deterministic continuous solution via ODE.",
+
+    ssa: "",
+
+    tauLeaping: "This algorithm calculates multiple reactions in a single step over a given tau step size.  The change in propensities over this step are bounded "+
+        "by bounding the relative change in state, yielding greatly improved run-time performance with very little trade-off in accuracy",
+
+    hybrid: "This algorithm uses a root-finding interpretation of the direct SSA method, along with ODE solvers to simulate ODE and Stochastic systems interchangeably "+
+        "or simultaneously.",
+
+    chooseForMe: "Hybrid will be chosen based on Event, Rules, and other SBML model components or if the model is represented as Concentration.  Tau Leaping will be "+
+        "chosen if the Tau Tolerance is changed to a value other than 0.03.  SSA will be chosen if the model is represented as Population."
+  },
+  parameterSweepSettings: {
+    sweepType: "The number of parameters to sweep through.",
```

Comment:
I feel like the variable name for this is slightly confusing.  I feel like "Type" and "number of..." don't mesh up in my brain well... maybe we should call this sweepDimensions?

Reply:
After looking at where this tooltip is, actually I think it is fine... it is obvious from the context of the page.